event/parsing/wrpParsers.go

```diff
@@ -24,7 +26,7 @@ func GetWRPBootTime(msg wrp.Message) (int64, error) {
 	if bootTimeStr, ok := msg.Metadata[bootTimeKey]; ok {
 		bootTime, err = strconv.ParseInt(bootTimeStr, 10, 64)
 		if err != nil {
-			return 0, err
+			return 0, emperror.WrapWith(errBootTimeParse, "error parsing boot-time", err)
```

Comment:
This doesn't match the function signature for `WrapWith`:
https://pkg.go.dev/emperror.dev/emperror#WrapWith

But I'm not sure you need `WrapWith()`, you could also do:
```
fmt.Errorf("%w: %v", errBootTimeParse, err)
```